.gitignore

```diff
@@ -17,3 +18,4 @@ nbproject
 # Other
 
 /build/
+/build_*/
```

Comment:
IDE specific notations were already present in the .gitignore file (there is a whole *"IDEs and editors"* section), so I don't really get the first part of this change request.

I added the `build_*` folder to the .gitignore file, because `build` was already there, but a separate folder for SQLite and PgSQL configuration was useful during the development process, since we had to test against both.
The `build` and `build_*` folders are not related to any IDE, but a developer-specific stuff, simply the most common names for the CMake build folders and are widely ignored in Git repositories.

Can be merged into a single line, will do it later.

Reply:
Fixed in 64cb407581662e78675baf916a95dba2b765b3e4.